### PR TITLE
User Guide: Fix redo command heading

### LINKED ISSUE #224: [PE-D][Tester B] Typo in UG redo command
Refer to this picture:

The subtitle now is:

Undoing a command: `redo`

However, it should be:

Redoing a command: `redo`

You did a good job in making the description here easy to be understood by users.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: KumChaiYin/ped#4

### PR DESCRIPTION
**Pull request by andrechuakj**
Closes #224

What have you done in this PR?
* Updated redo command heading to "Redoing a command"

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -258,7 +258,7 @@ Undoes the previous command that the user has entered, which has changed the dat
 
 Format: `redo`
 
-### Undoing a command: `redo`
+### Redoing a command: `redo`
```
Comment (SelwynAng) [inline docs/UserGuide.md:261]: I think it will be good to add in which specific commands can be redone (commands that change data within the JSON file).

**Review by SelwynAng — COMMENTED** [review]
LGTM!

File: docs/UserGuide.md (hunk 1 above)
Comment (andrechuakj) [reply docs/UserGuide.md:261]: Agreed, but I think we can add this in a separate PR

Comment (wr1159) [reply docs/UserGuide.md:261]: Okay if we're adding it in a separate PR, I think this is good to merge

**Review by wr1159 — APPROVED** [review]

**Pull request merged**